pkg/engine/lifecycletest/testdata/output/TestCanceledRefresh/diff.stdout.txt

```diff
@@ -1,12 +1,12 @@
 <{%reset%}>  pulumi:providers:pkgA: (same)
-<{%reset%}>    [id=7061565a-87e7-442b-8db1-b1e52e4aa003]
+<{%reset%}>    [id=5e487d6f-21a2-4212-bd9b-c0f2da35d10d]
 <{%reset%}><{%reset%}>    [urn=urn:pulumi:test::test::pulumi:providers:pkgA::default]
-<{%reset%}><{%fg 3%}>~ pkgA:m:typA: (update)
+<{%reset%}><{%reset%}>  pkgA:m:typA: (same)
 <{%reset%}>    [id=0]
 <{%reset%}><{%reset%}>    [urn=urn:pulumi:test::test::pkgA:m:typA::resA]
-<{%reset%}><{%fg 3%}>    --outputs:--<{%reset%}>
+<{%reset%}><{%reset%}>    --outputs:--<{%reset%}>
 <{%fg 2%}>  + foo: <{%reset%}><{%fg 2%}>"bar"<{%reset%}><{%fg 2%}>
 <{%reset%}><{%fg 13%}><{%bold%}>Resources:<{%reset%}>
-    <{%fg 3%}>~ 1 updated<{%reset%}>
+    1 unchanged
```

Comment:
This and other similar changes are intentional - the only change is in outputs so it doesn't report as a diff.

Note that these tests do the equivalent of `--show-sames`, so they actually do show the diff for the sames, which wouldn't be "normal" in practice.